zhmccli/_cmd_partition.py

```diff
@@ -218,6 +240,20 @@ def partition_create(cmd_ctx, cpc, **options):
               'image file on the HMC.')
 @click.option('--boot-iso', type=str, required=False,
               help='Boot from an ISO image mounted to this partition.')
+@click.option('--ssc-host-name', type=str, required=False,
+              help='Secure Service Container host name.')
+@click.option('--ssc-boot-selection',
+              type=click.Choice(['installer']), required=False,
+              help='On an Update request, this property can be set from '
+              '"appliance" to "installer" only')
```

Comment:
This help text can be improved to better say what it does. Also, the default is not "installer". The default is not to change it.

Proposal for the help text:

"Set the boot mode of the Secure Service Container to run the SSC Appliance Installer again upon next partition start. Default: No change."